blog/_posts/2016-08-05-Intro-Team-Reactives.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Reatives - Bond For a Bang ☼.☼!
+layout: post
+created_at: Fri Aug 05 2016
+permalink: blog/2016-08-05-Intro-Team-Reactives
+current: blog
+author: Shwetha & Tu An
```

Comment:
Can you add the line `twitter: teamreactives` under the author line? Our template for now requires a twitter handle, otherwise it shows a strange comma after your names in the post metadata. I'm working on improving this, but for now each post requires a twitter account in the frontmatter section.